web_extended_email_widget/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Extended Email Widget
+======================
+
+Features
+========
+
+* This module allows to change the default behaviour of email widgets with the possibility to add cc, bcc and subject fields.
+
+* It also adds the possibility to create a button (from the object with a client action) to send a email to specified addresses, that is useful in tree views or to send a email to all contacts
+
+Usage
+=====
+
+In the view declaration add an options attribute to an email widget field with the reference to the fields in view to include::
+
+    <field name="email" widget="email" options="{'cc': 'email_cc', 'subject': 'name'}/>
+
+This code will use the field email_cc from the view as the cc, and the field name to use as email subject.
+
+Allowed options are:
+
+* cc
+* bcc (May not work with all email clients)
+* replyto (May not work with all email clients)
+* subject
+
+For use the button its possible to declare a new function in the object that will be used::
```

Comment:
s/For use the button its possible to declare/For using the button, you have to declare